dom.go

```diff
@@ -849,14 +849,12 @@ func (b *batchRenderer) render(startTime float64) {
 		}
 
 		// Perform render.
-		prevRender := c.Context().prevRender
-		nextRender, skip, pendingMounts := renderComponent(c, prevRender)
+		prevHTML := extractHTML(c.Context().prevRender)
+		nextHTML, skip, pendingMounts := renderComponent(c, c.Context().prevRender)
```

Comment:
`prevHTML` and `nextHTML` are resolved recursively, so that the first `*HTML` result is returned from any depth of nested components.  The components track their children through `Core.prevRender` as normal, and bubble the `*HTML` up to the outer component.